packages/framework/src/Resources/config/services.yml

```diff
@@ -973,10 +973,10 @@ services:
 
     Shopsys\FrameworkBundle\Component\Redis\RedisFacade:
         arguments:
-            $cacheClients:
-            - '@snc_redis.bestselling_products'
-            - '@snc_redis.doctrine_metadata'
-            - '@snc_redis.doctrine_query'
+            $allClients: !tagged snc_redis.client
+            $persistentClients:
+                - '@snc_redis.global'
```

Comment:
from the view of RedisFacade it is persistentClient but from the view of `RedisVersionsFacade` it is not.

Reply:
From the point of view of `RedisVersionsFacade`, it's still `$globalClient` and the facade knows what should get removed and what should not. Maybe `$persistentClient` is not a good name - do you have any suggestions?

Reply:
i do not.